app/models/review_request.rb

```diff
@@ -0,0 +1,14 @@
+class ReviewRequest < ApplicationRecord
+  extend Enumerize
+
+  belongs_to :pull_request, inverse_of: :review_requests
+  belongs_to :reviewee, inverse_of: :review_requests
+  belongs_to :reviewer, inverse_of: :review_requests
+
+  validates :name, presence: true, lenght: { maximum: 1000 }
```

Comment:
__[Misspell]__

"lenght" is a misspelling of "length"


<!-- Comment by [SideCI](https://sideci.com) -->